File: 263. Ugly Number/263. Ugly Number.js
/**
 * @param {number} num
 * @return {boolean}
 */
var isUgly = function(num) {
    if(num === 0) return false;
    
    for(var i = 2; i < 6; i++) {
        while(num % i === 0) {
            num = num / i
        }
    }
    
    return num === 1;
};

File: 263. Ugly Number/263. Ugly Number.rb
# @param {Integer} num
# @return {Boolean}
def is_ugly(num)
	return false if num == 0
	
   (2..5).each do |i|
   	while num % i == 0
   		num = num / i
   	end
   end
   num == 1
end
